tests/tracking/context/test_databricks_job_context.py

```diff
@@ -23,7 +23,7 @@ def test_databricks_job_run_context_tags():
     patch_job_type = mock.patch("mlflow.utils.databricks_utils.get_job_type")
     patch_webapp_url = mock.patch("mlflow.utils.databricks_utils.get_webapp_url")
 
-    with patch_job_id as job_id_mock, patch_job_run_id as job_run_id_mock, patch_job_type as job_type_mock, patch_webapp_url as webapp_url_mock:  # noqa
+    with patch_job_id as job_id_mock, patch_job_run_id as job_run_id_mock, patch_job_type as job_type_mock, patch_webapp_url as webapp_url_mock:  # pylint: disable=line-too-long
```

Comment:
Same for this file.

Reply:
@liangz1 

We can format this line like below with `fmt: off/on` to disable black formatting:

```suggestion
    # fmt: on
    with patch_job_id as job_id_mock, \
         patch_job_run_id as job_run_id_mock, \
         patch_job_type as job_type_mock, \
         patch_webapp_url as webapp_url_mock:
        # fmt: off
```

Reply:
Yes, I feel this is easier to read. Thanks!

Reply:
@liangz1 It turns out this doesn't work. Instead, we need to do wrap the entire with block:

```python
    # fmt: off
    with patch_job_id as job_id_mock, \
         patch_job_run_id as job_run_id_mock, \
         patch_job_type as job_type_mock, \
         patch_webapp_url as webapp_url_mock:
        assert DatabricksJobRunContext().tags() == {
            MLFLOW_SOURCE_NAME: "jobs/{job_id}/run/{job_run_id}".format(
                job_id=job_id_mock.return_value, job_run_id=job_run_id_mock.return_value
            ),
            MLFLOW_SOURCE_TYPE: SourceType.to_string(SourceType.JOB),
            MLFLOW_DATABRICKS_JOB_ID: job_id_mock.return_value,
            MLFLOW_DATABRICKS_JOB_RUN_ID: job_run_id_mock.return_value,
            MLFLOW_DATABRICKS_JOB_TYPE: job_type_mock.return_value,
            MLFLOW_DATABRICKS_WEBAPP_URL: webapp_url_mock.return_value,
        }
    # fmt on
```

Reply:
Can we use `pylint: disable` here? I just don't like wrapping unrelated code with `# fmt off/on`